packages/mobile/src/fiatExchanges/Simplex.tsx

```diff
@@ -0,0 +1,66 @@
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { WebView } from 'react-native-webview'
+import { useSelector } from 'react-redux'
+import AccountNumber from 'src/components/AccountNumber'
+import { SIMPLEX_URI } from 'src/config'
```

Comment:
AFAIK, this new config wasn't committed, causing the build to fail.

Reply:
Updated, thanks for catching this